go.mod

```diff
@@ -38,7 +38,6 @@ require (
 	github.com/creack/pty v1.1.18
 	github.com/crewjam/saml v0.4.12
 	github.com/datastax/go-cassandra-native-protocol v0.0.0-20220706104457-5e8aad05cf90
-	github.com/denisenkom/go-mssqldb v0.11.0 // replaced
```

Comment:
Please update .github/dependabot.yml, removing `denisenkom` from the list and adding the microsoft dep in its place. (Also keep that list in alphabetical order.)